linter.py

```diff
@@ -1,26 +1,14 @@
-#
-# linter.py
-# Linter for SublimeLinter3, a code checking framework for Sublime Text 3
-#
-# Written by Jon Surrell
-# Copyright (c) 2014 Jon Surrell
-#
-# License: MIT
-#
-
-"""This module exports the Gotype plugin class."""
-
 from os.path import dirname
 from SublimeLinter.lint import Linter, util
 
 
 class Gotype(Linter):
-    """Provides an interface to gotype."""
-
-    syntax = ('go', 'gosublime-go')
     executable = "gotype"
     regex = r'(?P<filename>^.+):(?P<line>\d+):(?P<col>\d+):\s+(?P<message>.+)'
     error_stream = util.STREAM_STDERR
+    defaults = {
+        'selector': 'source.go'
+    }
 
     def __init__(self, view, syntax):
```

Comment:
Remove __init__ completely, user can set env at will.

executable and executable_path are deprecated

Reply:
Like so? 5ed938b